src/App/OrganisasjonerOgTilgangerProvider.tsx

```diff
@@ -167,24 +167,6 @@ export const OrganisasjonerOgTilgangerProvider: FunctionComponent = (props) => {
                     tidMs,
                 });
             },
-            hentUserInfo()
-                .then(({ organisasjoner, tilganger, altinnError }) => {
-                    if (altinnError) {
-                        setVisFeilmelding(true);
-                        addAlert('TilgangerAltinn');
-                    }
-                    setAltinnorganisasjoner(organisasjoner);
-                    setAltinntilganger(
-                        Record.fromEntries(tilganger.map((it) => [it.id, Set(it.organisasjoner)]))
-                    );
-                })
-                .catch((error) => {
-                    Sentry.captureException(error);
-                    setAltinntilganger(Record.map(altinntjeneste, () => Set()));
```

Comment:
litt usikker på hvordan bevare denne oppførselen. Har ike noe hard feil lenger. Må kanskje sette et flagg i fallbackdata, eller propagere harde feil som error fra hooken

Reply:
Tenker det er litt viktig å kunne kommunisere med brukere at vi har gitt opp å laste siden 🤔 Ideelt sett, så hadde vel `useUserInfo` på et eller annet vis sagt i fra at den har gitt opp. 

Kan det være at else-branchen i `onErrorRetry` kan kaste erroren oppover? Da kastes den først når vi har gitt opp. For da antar jeg at `error` i `const { data, error } = useSWR(...)` vil være undefined før `onErrorRetry` faktisk kaster erroren, og ikke når den svelger den. Mulig det ikke funker, da.

Reply:
når OnErrorRetry er exhausted så returneres error. Man trenger ikke å kaste videre. Kan være det henger sammen med at man ikke callbacker på revalidate

Reply:
Var litt det jeg mente med å sette et flagg i fallbackdata. Men man kunne evt hatt en result type som enten er feil eller userInfo.

Reply:
803c0099679d9de2c91d1b2c2a0d56554889a628